### PR TITLE
Emotion make styles batch2

### PR DESCRIPTION
**Pull request by EstherDarkish**
Emotion migration, new batch of files.

### CODE REVIEW

**Review by Tristan-WorkGH — COMMENTED** [review]
seem good to me, just a point i'm not sure and cannot found the doc about

File: src/components/app-top-bar.js [hunk 1]
```diff
@@ -67,16 +66,16 @@ import {
 } from './utils/equipment-types';
 import { fetchAppsAndUrls } from '../services/utils';
 
-const useStyles = makeStyles((theme) => ({
+const styles = {
     tabs: {
         flexGrow: 1,
     },
-    label: {
+    label: (theme) => ({
```
[the remark this answers is not in the document]
Comment (Tristan-WorkGH) [reply src/components/app-top-bar.js:73]: @archendel do you know something?

[the remark this answers is not in the document]
Comment (EstherDarkish) [reply src/components/app-top-bar.js:73]: Yes, it works.

File: src/components/spreadsheet/columns-config.js [hunk 2]
```diff
@@ -239,14 +236,10 @@ export const ColumnsConfig = ({
     const renderColumnConfigLockIcon = (value) => {
         if (selectedColumnsNames.has(value)) {
             if (lockedColumnsNames.has(value)) {
-                return <LockIcon className={classes.columnConfigClosedLock} />;
+                return <LockIcon sx={styles.columnConfigClosedLock} />;
             } else {
                 if (lockedColumnsNames.size < MAX_LOCKS_PER_TAB) {
```
[the remark this answers is not in the document]
Comment (EstherDarkish) [reply src/components/spreadsheet/columns-config.js:241]: That's not the goal of this PR, so I will not change it

[the remark this answers is not in the document]
Comment (Tristan-WorkGH) [reply src/components/spreadsheet/columns-config.js:241]: out of scope

File: src/components/app-top-bar.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tristan-WorkGH) [reply src/components/app-top-bar.js:73]: found the answer : it's not from emotion but from mui (see https://mui.com/system/getting-started/the-sx-prop/#palette)
```javascript
<Box sx={{ bgcolor: 'primary.main' }} />
// equivalent to backgroundColor: theme => theme.palette.primary.main
```

**Pull request merged**